lib/service/logger.js

```diff
@@ -14,19 +14,23 @@
 
 const isNil = require('lodash.isnil');
 
+const util = require('util');
+
 function _log(level, message) {
   if (isNil(message)) return;
   if (message instanceof Error) message = message.toString();
 
-  const logMessage = JSON.stringify({
+  const logMessage = {
     message,
     level,
     dlcId: process.env.DLC_ID,
     backingServerId: process.env.BACKING_SERVER_ID
-  });
+  };
+
+  const logMessageString = `${util.format(JSON.stringify(logMessage))}\n`;
```

Comment:
A few questions:
1. What is `util.format` used for here?
2. Both util.format and JSON.stringify output strings -- why wrap it again in a string?